### ISSUE TITLE
Error on update (pyrtable==0.7.10)

### ISSUE THREAD

**Issue by vhmartinezm** [vilarneto/pyrtable#3]
save method works but I'm getting an error on update:

```
class Devices(MyBaseRecord):
    class Meta:
        table_id = 'Devices'

    name = StringField('Name')
    description = StringField('Description')


class Ips(MyBaseRecord):
    class Meta:
        table_id = 'Ips'

    id = IntegerField('Id')
    internal = StringField('Internal')
    external = StringField('External')
    device = SingleRecordLinkField('Device', linked_class=Devices)


device = Devices(name="pc", description="My pc")
device.save()

ip = Ips(internal="192.168.100.15", external="181.143.142.160", device=device)
ip.save() # It works

ip.internal = "192.168.100.16"
ip.save() # Error

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-50-7a5a8299798b> in <module>
----> 1 ip.save()

/usr/local/lib/python3.9/site-packages/pyrtable/record.py in save(self)
    214         """
    215         from pyrtable.context import get_default_context
--> 216         get_default_context().save(self.__class__, self)
    217
    218     def encode_to_airtable(self, include_non_dirty_fields=False) -> Dict[str, Any]:

/usr/local/lib/python3.9/site-packages/pyrtable/context/base.py in save(self, record_cls, record)
    158             self._create(record_cls, record)
    159         else:
--> 160             self._update(record_cls, record)
    161
    162     def delete_id(self, *,

/usr/local/lib/python3.9/site-packages/pyrtable/context/base.py in _update(self, record_cls, record)
    141             if 400 <= response.status_code < 500:
    142                 error = response.json().get('error', {})
--> 143                 error_message = error.get('message', '')
    144                 error_type = error.get('type', '')
    145

AttributeError: 'str' object has no attribute 'get'
```

**Comment by vhmartinezm** [comment]
same on delete method

**Comment by vilarneto** [comment]
Hello, does it work if you replace the name of the `id` field on `Ips`?
```python
class Ips(MyBaseRecord):
    class Meta:
        table_id = 'Ips'

    identifier = IntegerField('Id')  # <- changed here
    internal = StringField('Internal')
    external = StringField('External')
    device = SingleRecordLinkField('Device', linked_class=Devices)
```
That's because `<record_object>.id` holds an internal identifier provided by Airtable (sorry – it's not clear in the documentation).

If that works I will implement a sanity check.

**Comment by vhmartinezm** [comment]
It works! no problem. Thank you!

**Comment by vilarneto** [comment]
Sanity check & improved documentation implemented in c97ec2d.
